translations/de.i18n.json

```diff
@@ -7,6 +7,7 @@
       "offline_body": "Video wird abgespielt, sobald die Übertragung beginnt",
       "http_problem_title": "Etwas ist schief gelaufen",
       "http_problem_body" : "Versuchen Sie, die Seite zu aktualisieren",
+      "no_longer_live": "Broadcast is no longer live",
```

Comment:
does it correct translate?

Reply:
@semarche-kaltura 
no, but we still waiting for the translation from the translation company
i'll update this when we have the right translation

Reply:
if no - remove the translate key from de.i18n.json file. By default uses value from `<Text id="kalturaLive.no_longer_live">**Broadcast is no longer live**</Text>`